libV2/helpers/collection/generateSkeletionTreeFromOpenAPI.js

```diff
@@ -26,13 +27,23 @@ let _ = require('lodash'),
       meta: {}
     });
 
-    _.forEach(openapi.paths, function (methods, path) {
-      let pathSplit = path === '/' ? [path] : _.compact(path.split('/'));
+    /**
+     * Get all the paths sorted in desc order.
+     */
+    const paths = Object.keys(openapi.paths).sort((a, b) => {
+      return (a > b ? -1 : 1);
```

Comment:
@sujayvenaik Shouldn't we sort based on the number of segments a particular path have ?